src/main/java/com/netchill/services/serie/SerieService.java

```diff
@@ -1,22 +1,37 @@
 package com.netchill.services.serie;
 
-import com.netchill.api.moviedb.MovieDbApiClient;
+import com.coreoz.plume.jersey.errors.WsException;
+import com.netchill.api.moviedb.models.MovieDbPaginatedResponse;
 import com.netchill.api.moviedb.models.Production;
-import com.netchill.api.moviedb.models.Serie;
+import com.netchill.api.moviedb.services.serie.SerieApiService;
+import com.netchill.webservices.error.NetchillWsError;
 
 import javax.inject.Inject;
 import javax.inject.Singleton;
+import java.util.List;
+import java.util.Optional;
 
 @Singleton
 public class SerieService {
-    private MovieDbApiClient movieDbApiClient;
+    private SerieApiService serieApiClient;
 
     @Inject
-    private SerieService(MovieDbApiClient movieDbApiClient) {
-        this.movieDbApiClient = movieDbApiClient;
+    private SerieService(SerieApiService serieApiClient) {
+        this.serieApiClient = serieApiClient;
     }
 
-    public Production getSerieById(Long id) {
-        return this.movieDbApiClient.getSerieById(id);
+    public List<Production> getTopNetflixOriginals() {
+        return this.serieApiClient.getNetflixOriginals().getResults();
+    }
+
+    public MovieDbPaginatedResponse<Production> getAllNetflixOriginals(Integer page) {
+        return this.serieApiClient.getNetflixOriginals(page);
+    }
+
+    public Optional<Production> getSerieById(Long id) {
+        return Optional.ofNullable(
+                Optional.of(this.serieApiClient.getSerieById(id))
+                        .orElseThrow(()-> new WsException(NetchillWsError.RESOURCE_NOT_FOUND))
+        );
     }
```

Comment:
pk 2 optional.

Optional. ofNullable => la valeur peut être null.
Optional. of => Si la valeur est null ça lève une exception.

Optional. ofNullable(null) OK
Optional. of(null) KO

Reply:
exception à déplacer dans le WS.